doc/man/parrot_namespace.m4

```diff
@@ -0,0 +1,60 @@
+include(manual.h)dnl
+HEADER(parrot_namespace)
+
+SECTION(NAME)
+BOLD(parrot_namespace) - run a command in a modified namespace.
+
+SECTION(SYNOPSIS)
+CODE(BOLD(parrot_cp [options] PARAM(command)))
+
+SECTION(DESCRIPTION)
+
+CODE(parrot_namespace) detects whether it is already running under Parrot
+and either forks a new mount namespace in the existing Parrot session or
+simply executes CODE(parrot_run). For applications that only need to make
+mount-related changes, CODE(parrot_namespace) is a drop-in replacement
+for CODE(parrot_run) that automatically handles nested invocations.
+
+SECTION(OPTIONS)
+
+OPTIONS_BEGIN
+OPTION_ITEM(`-M, --mount /foo=/bar')Mount (redirect) CODE(/foo)
+to CODE(/bar) (CODE(PARROT_MOUNT_STRING))
+OPTION_ITEM(`-m, --ftab-file <path>')Use CODE(<file>) as a
+mountlist (CODE(PARROT_MOUNT_FILE))
+OPTION_TRIPLET(-l, ld-path, path)Path to ld.so to use.
```

Comment:
@trshaffer , does the environment variable `PARROT_LDSO_PATH` work for this purpose?